en/docs/security-announcements/cve-justifications/2025/CVE-2021-42646.md

```diff
@@ -0,0 +1,36 @@
+---
+title: CVE-2021-42646
+category: security-announcements
+date: June 02, 2025
+---
+
+# CVE-2021-42646
+
+<p class="doc-info">WSO2 Products Impacted: No</p>
+<p class="doc-info">Customers Actions Required: No</p>
+---
+
+### REPORTED VULNERABILITY
+An XML External Entity (XXE) vulnerability in the file-based service provider creation feature of the WSO2 Management Console allows attackers to send crafted GET requests to access sensitive files or cause denial of service by exploiting insecure XML parsing [^1].
+
+### REPORTED PRODUCTS
+- WSO2 API Manager: 2.6.0, 3.0.0, 3.1.0, 3.2.0, 4.0.0  
+- WSO2 Identity Server: 5.7.0, 5.8.0, 5.9.0, 5.10.0, 5.11.0  
+- WSO2 Identity Server as Key Manager: 5.7.0, 5.9.0, 5.10.0
+
+### WSO2 JUSTIFICATION
+The vulnerability mentioned in CVE-2021-42646 has been addressed and fixed in the respective versions as per the advisory published at WSO2-2021-1289 [^2].
+
+### CONCLUSION
+
+Based on the following key points:
+
+- The incorrect listing of `org.wso2.carbon.identity.framework:org.wso2.carbon.identity.application.mgt` as a vulnerable component.
```

Comment:
@Tharanidk How we say this? According to the public PR attached in the advisory (https://security.docs.wso2.com/en/latest/security-announcements/security-advisories/2021/WSO2-2021-1289/#solution), component name isn't the issue here I think. Should be the incorrect version information since the scanners were not aware of the support versioning. 

If so, it's better if we can list down the component versions like in https://security.docs.wso2.com/en/latest/security-announcements/cve-justifications/2024/CVE-2022-29464/#affected-components-and-fixed-versions. Otherwise, having the same information as the public advisory in the cve-justification wouldn't make much difference.

Reply:
Could you please add the unaffected ranges as we don't have permission to check those